sdks/python/apache_beam/transforms/combinefn_lifecycle_test.py

```diff
@@ -69,6 +69,7 @@ def test_non_liftable_combine(self):
 
   @skip_unless_v2
   def test_combining_value_state(self):
+    self.skipTest('BEAM-11793')
```

Comment:
Should this only skip for DataflowRunner?

Reply:
skip_unless_v2 will only make it valid for dataflow runner.

Reply:
hmm I guess not.

Reply:
My reading of that logic is that it skips the test for when DataflowRunner is used with runner v2 specified. So it should run the test for any other runner, right?

Reply:
Added the check.